project/AppDependencies.scala

```diff
@@ -7,9 +7,9 @@ object AppDependencies {
   private val circeVersion = "0.12.0"
 
   val compile = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "2.25.0",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "3.0.0",
     "uk.gov.hmrc" %% "play-frontend-hmrc"         % "0.20.0-play-27",
-    "uk.gov.hmrc" %% "play-frontend-govuk"        % "0.51.0-play-27",
+    "uk.gov.hmrc" %% "play-frontend-govuk"        % "0.53.0-play-27",
```

Comment:
This combination of HMRC and GOVUK isn't tested/recommended, but nor is the previous combination.  We currently recommend:

`play-frontend-hmrc 0.21.0` and `play-frontend-hmrc 0.49.0`

Reply:
That reminds me, we should update `govuk-frontend` in `hmrc-frontend`.